addon/utils/model-factory.js

```diff
@@ -151,45 +152,55 @@ export function getSubFormsForNode(
 
 /**
  * Returns all the children (fields & property-groups) for the given property-group, in order.
+ * @deprecated use #getChildrenForSection instead
  *
  * @returns list of children for the given property-group, in order.
  */
 export function getChildrenForPropertyGroup(
   group,
   { form, store, graphs, node }
 ) {
+  return getChildrenForSection(group, { form, store, graphs, node });
+}
+
+/**
+ * Returns all the children (fields & sections) for the given section, in order.
+ *
+ * @returns list of children for the given section, in order.
+ */
+export function getChildrenForSection(section, { form, store, graphs, node }) {
+  const formGraph = graphs.formGraph;
   const conditionals = fieldsForForm(form, {
     store,
-    formGraph: graphs.formGraph,
+    formGraph,
     sourceGraph: graphs.sourceGraph,
     metaGraph: graphs.metaGraph,
     sourceNode: node,
   });
 
-  // NOTE: contains all children for a property-group (this can include other nested property-groups)
-  const children = store
-    .match(undefined, SHACL('group'), group.uri, graphs.formGraph)
+  // NOTE: contains all children for a section (this can include other nested sections)
+  const children = sectionHelpers
+    .getItems(section.uri, { store, formGraph })
```

Comment:
nitpick: you refer to `getItems` here but they will be stored in the `const children` should we rename the helper method to `getChildren()`?